Add `mkosi.bump` script (#2905)
systemd/mkosi: mkosi/resources/mkosi.md

    `/usr/lib/os-release` or similar, in particular the `IMAGE_VERSION=`
    field of it).

`ImageVersionFile=`, `--image-version-file=`
:   Takes a path to a file to read the `ImageVersion=` from if not
    explicitly set. The file will also be used when running `mkosi bump`
    or when building with the `--auto-bump` option enabled. This value

>     default to `mkosi.version` if it is found in the config directory.

Reviewer: I'm not a big fan of a separate setting for this. I'd prefer to just make `mkosi.version` the default path for `ImageVersion=`

Author: Dropped the second commit.

  artifacts from `SplitArtifacts=yes` or RPMs built in a build script).
  Note that this script does not use the tools tree even if one is configured.

* If **`mkosi.bump`** (`BumpScript=`) exists, it is executed when running
  `mkosi bump` or after a successful build with the `--auto-bump` option.
  A bump script can customize the logic used when incrementing from one version

>   to a newer version. The current version is passed as first argument to the
>   script and the new version to bump to is expected to be output to stdout.

Reviewer 2: ```suggestion
  to a newer version. The current version is passed as first and only argument to the
  script and the new version is expected on stdout.
```
